dev-docs/bidders/frvradn.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: bidder
+title: FRVR Ad Network
+description: FRVR Ad Network Prebid Bidder Adapter
+biddercode: frvradn
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
```

Comment:
If you support multiple formats we recommend to set the `multiformat_supported` meta data to one of the following values `will-bid-on-any`, `will-bid-on-one`, `will-not-bid`.

Same goes for `safeframes_ok`: `true` or `false`